##duration of experiment  in seconds 
experiment_duration: 25


